cmd/tendermint/commands/testnet.go

```diff
@@ -64,6 +65,8 @@ func init() {
 		"P2P Port")
 	TestnetFilesCmd.Flags().BoolVar(&randomMonikers, "random-monikers", false,
 		"Randomize the moniker for each generated node")
+	TestnetFilesCmd.Flags().StringVar(&rpcListenAddr, "rpc-laddr", "tcp://127.0.0.1:26657",
```

Comment:
I'm against of adding any more options to testnet cmd. We should just pass the whole config to it in tests instead of adding config variables one by one.

Reply:
Sure, I'll take it out.